lib/Runtime/Library/ByteCodeString.h

```diff
@@ -0,0 +1,21 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace Js
+{
+    class ByteCodeString : public JavascriptString
+    {
+    private:
+        Field(Utf8SourceInfo*) utf8SourceInfo;
+
+    protected:
+        ByteCodeString(StaticType* type, const char16* content, charcount_t charLength, Utf8SourceInfo* utf8SourceInfo);
```

Comment:
you're not doing SAL annotations on this?  I see them on `NewWithByteCodeBuffer` below.

Reply:
I'll take a look, there weren't any annotations on LiteralString (which this is based on), so I didn't notice it.